<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generateWidget</class>
 <widget class="QWidget" name="generateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generater</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(200, 200, 200);</string>
  </property>
  <widget class="QPushButton" name="generateButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background-color: rgb(255, 255, 255,0);
	border: 2px solid rgb(0,0,0);
	border-radius: 20px;
	color: rgb(0,0,0);
	font: 75 11pt &quot;等线&quot;;
}
QWidget:hover
{
	font-size:12pt;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}
QWidget:pressed 
{
	font-size:9pt;
}</string>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>300</width>
     <height>461</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background-color: rgb(60,60,60);
	border: 0px solid rgb(0,0,0);
	border-radius: 7px;
	color: #fff;
	font:75 8pt &quot;等线&quot;;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'等线'; font-size:8pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>401</width>
     <height>321</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background-color: rgb(255, 255, 255,0);
}
QWidget.QLineEdit
{
	background-color: rgb(60,60,60);
	color: #fff;
	font: 9pt &quot;等线l&quot;;
	border:0px solid rgb(0,0,0);
	border-radius:5px;
	padding-left:5px;
}
QWidget.QLineEdit:focus
{
	background-color: rgb(60,60,60,0);
	color: #000;
	border:2px solid rgb(0,0,0);
	border-radius:5px;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);
font: 87 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>source</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);
font: 87 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>target</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);
font: 87 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>duration</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="durationEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);
background-color: rgb(255, 255, 255,0);
font: 75 9pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255,255,255,0);
color: rgb(0,0,0);
font: 87 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>tag</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sourceEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tagEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sourceButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	background-color:rgb(40,40,40,0);
	font: 9pt &quot;Arial&quot;;
}
QWidget:hover
{
	font-size:12pt;
}
</string>
    </property>
    <property name="text">
     <string>📂</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="targetButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	background-color:rgb(40,40,40,0);
	font: 9pt &quot;Arial&quot;;
}
QWidget:hover
{
	font-size:12pt;
}
</string>
    </property>
    <property name="text">
     <string>📂</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
